### ISSUE TITLE
False positive in possible_missing_comma

### ISSUE THREAD

**Issue by YaLTeR** [rust-lang/rust-clippy#4399]
`clippy 0.0.212 (e3cb40e 2019-06-25)` and also the nightly `clippy 0.0.212 (72da101 2019-08-11)`

Looks like a similar case to #3244 and #3396.

We have code like this:
```rust
  let hfilter: [i32; 7] = [
    coeffs[1][0] as i32,
    coeffs[1][1] as i32,
    coeffs[1][2] as i32,
    128
      - 2 * (coeffs[1][0] as i32 + coeffs[1][1] as i32 + coeffs[1][2] as i32),
    coeffs[1][2] as i32,
    coeffs[1][1] as i32,
    coeffs[1][0] as i32,
  ];
```
That expression starting with `128` got broken into two lines by `rustfmt` as it doesn't fit into a single line, and now `clippy` is reporting a `possible_missing_comma` there.
